docs/awesome-pets/papers/tools/mpc.md

```diff
@@ -47,6 +47,10 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Ian McQuoid, Mike Rosulek, Lawrence Roy*  
   AsiaCrypt 2021, [eprint](https://eprint.iacr.org/2021/682), MRR21
 
+- Ferret: Fast Extension for Correlated OT with Small Communication
```

Comment:
Can you add to spaces at the end of this line? It'll make it look nicer on Github